addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -45,24 +48,44 @@ def unlink(self):
                 # Note: invitation depends on field `rtc_inviting_session_id` so the cancel must be
                 # done before the delete to be able to know who was invited.
                 channel._rtc_cancel_invitations()
+                # If there is no member left in the RTC call, we remove the SFU channel uuid as the SFU
+                # server will timeout the channel. It is better to obtain a new channel from the SFU server
+                # than to attempt recycling a possibly stale channel uuid.
+                channel.sfu_channel_uuid = False
         notifications = [(channel, 'discuss.channel/rtc_sessions_update', {
             'id': channel.id,
             'rtcSessions': [('DELETE', [{'id': session_data['id']} for session_data in sessions_data])],
         }) for channel, sessions_data in self._mail_rtc_session_format_by_channel().items()]
+        disconnect_requests = {}
         for rtc_session in self:
+            if rtc_session.channel_id.rtc_server_url:
+                disconnect_requests.setdefault(rtc_session.channel_id, []).append(rtc_session.id)
             target = rtc_session.guest_id or rtc_session.partner_id
             notifications.append((target, 'discuss.channel.rtc.session/ended', {'sessionId': rtc_session.id}))
         self.env['bus.bus']._sendmany(notifications)
-        return super().unlink()
+        return_value = super().unlink()
+        if self.env.context.get('notify_rtc_server'):
+            try:
+                for channel, ids in disconnect_requests.items():
+                    url = channel._get_rtc_server_url()
+                    sfu_channel_uuid = channel.sfu_channel_uuid
+                    if not url or not sfu_channel_uuid:
+                        continue
+                    private_key, _ = jwt.get_sfu_keys(self.env)
+                    requests.post(url + '/disconnect', json={"jsonWebToken": jwt.sign({'session_ids': ids, 'sfu_channel_uuid': sfu_channel_uuid}, key=private_key, ttl=20)})
```

Comment:
Most of the time it is not used, because as you said there is a timeout in cases of client-initiated disconnections (clean or not). But this route is used for when the server wants to force the disconnect a session (moderation feature).

Reply:
OK, I wasn't aware of the moderation feature.